toolchain/check/import_ref.cpp

```diff
@@ -409,42 +437,154 @@ class ImportRefResolver {
     return context_.inst_blocks().AddCanonical(contents);
   }
 
-  // Gets a local version of an imported generic.
-  auto GetLocalGeneric(SemIR::GenericId generic_id) -> SemIR::GenericId {
+  // Gets an incomplete local version of an imported generic. Most fields are
+  // set in the second pass.
+  auto GetIncompleteLocalGeneric(SemIR::InstId decl_id,
```

Comment:
FWIW, in other spots we do MakeIncompleteClass. "Get" doesn't imply creation for me.

Reply:
Done.